rotkehlchen/globaldb/upgrades/v9_v10.py

```diff
@@ -0,0 +1,73 @@
+    from rotkehlchen.db.upgrade_manager import DBUpgradeProgressHandler
+
+
+@enter_exit_debug_log(name='globaldb v9->v10 upgrade')
+def migrate_to_v10(connection: 'DBConnection', progress_handler: 'DBUpgradeProgressHandler') -> None:  # noqa: E501
+    """This globalDB upgrade does the following:
+
+    1. Adds main_asset column to asset_collections table.
+
+    This upgrade took place in v1.37
+    """
+
+    @progress_step('Adding main_asset column to asset_collections')
+    def add_main_asset_column(write_cursor: 'DBCursor') -> None:
+        write_cursor.execute('CREATE TABLE multiasset_mappings_backup AS SELECT * FROM multiasset_mappings')  # noqa: E501
+        write_cursor.execute("""
+            CREATE TABLE asset_collections_new(
+                id INTEGER PRIMARY KEY,
+                name TEXT NOT NULL,
+                symbol TEXT NOT NULL,
+                main_asset TEXT NOT NULL UNIQUE,
+                FOREIGN KEY(main_asset) REFERENCES assets(identifier) ON UPDATE CASCADE ON DELETE CASCADE,
+                UNIQUE(name, symbol)
+            );
+        """)  # noqa: E501
+        write_cursor.execute("""
+            INSERT INTO asset_collections_new
+            SELECT DISTINCT
+                m1.collection_id as id,
+                (SELECT name FROM asset_collections WHERE id = m1.collection_id) as name,
+                (SELECT symbol FROM asset_collections WHERE id = m1.collection_id) as symbol,
+                CASE
+                    WHEN collection_id = 23 THEN (
+                        SELECT asset
+                        FROM multiasset_mappings
+                        WHERE collection_id = 23
+                        AND asset = 'eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F'
+                    )
+                    ELSE (
+                        SELECT MIN(asset)
+                        FROM multiasset_mappings m2
+                        WHERE m2.collection_id = m1.collection_id
+                    )
+                END as main_asset
+            FROM multiasset_mappings m1
+        """)
+        write_cursor.execute('DROP TABLE asset_collections')
+        write_cursor.execute('ALTER TABLE asset_collections_new RENAME TO asset_collections')
+        write_cursor.execute('DROP TABLE multiasset_mappings')
+        write_cursor.execute("""
+            CREATE TABLE multiasset_mappings(
+                collection_id INTEGER NOT NULL,
+                asset TEXT NOT NULL,
+                FOREIGN KEY(collection_id) REFERENCES asset_collections(id) ON UPDATE CASCADE ON DELETE CASCADE,
+                FOREIGN KEY(asset) REFERENCES assets(identifier) ON UPDATE CASCADE ON DELETE CASCADE,
+                PRIMARY KEY(collection_id, asset)
+            )
+        """)  # noqa: E501
+        write_cursor.execute('INSERT INTO multiasset_mappings SELECT * FROM multiasset_mappings_backup')  # noqa: E501
+        write_cursor.execute('DROP TABLE multiasset_mappings_backup')
```

Comment:
As far as I can understand you just wrote the new upgrade over the old? Wtf? There is no longer any change to 
multiasset_mappings. Why are you doing this?
```suggestion
```